builtin/credential/aws-kms/path_config_user.go

```diff
@@ -0,0 +1,64 @@
+package awsKms
+
+import (
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigUser() *framework.Path {
+	return &framework.Path{
+		Pattern: "config/user",
+		Fields: map[string]*framework.FieldSchema{
+			"access_key": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Access key with kms:Decrypt permission.",
+			},
+
+			"secret_key": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Secret key with kms:Decrypt permission.",
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.WriteOperation: pathConfigRootWrite,
+		},
+
+		HelpSynopsis:    pathConfigRootHelpSyn,
+		HelpDescription: pathConfigRootHelpDesc,
+	}
+}
+
+func pathConfigRootWrite(
+	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
pathConfigRootWrite is mapped to WriteOperation on path 24. When the framework calls the function, it expects function to have that signature.

Though, admittedly I'm not exactly sure when the Vault wants logical.ErrorResponse and when it wants error to be populated.